src/Popover/Popover.Component.js

```diff
@@ -163,6 +163,24 @@ export class PopoverComponent extends React.Component {
                     </div>
                 </Example>
 
+                <Example
+                    centered
+                    title='Disable Edge Detection'>
+                    <div className='fd-doc__margin--popover'>
+                        <Popover
+                            body={bodyContent}
+                            control={<Button glyph='navigation-up-arrow' option='light' />}
+                            disableEdgeDetection='true'
```

Comment:
For the example we can just put `disableEdgeDetection` instead of `disableEdgeDetection='true'`, since it's boolean it's presence is the list makes it true